examples/nft-nd/nft.go

```diff
@@ -230,7 +230,7 @@ func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
 	}
 
 	tokenHash := crypto.Ripemd160(tokIn)
-	token := std.Base58Encode(tokenHash)
+	token := std.Base64Encode(tokenHash)
```

Comment:
I wanted it to be more user-friendly, but if we can't provide that then let's use something that works.